requirements.txt

```
@@ -1,2 +1,2 @@
-ansible>=2.10.0
+ansible>=2.10.0,<10
```

```suggestion
ansible>=6,<10
```
Really, we should also replace `>=2.10` since that value exists because of the previous requirement for `ansible-core`.